selfdrive/car/interfaces.py

```diff
@@ -141,10 +141,11 @@ def configure_torque_tune(candidate, tune, steering_angle_deadzone_deg=0.0, use_
 
     tune.init('torque')
     tune.torque.useSteeringAngle = use_steering_angle
-    tune.torque.kp = 1.0 / params['LAT_ACCEL_FACTOR']
-    tune.torque.kf = 1.0 / params['LAT_ACCEL_FACTOR']
-    tune.torque.ki = 0.1 / params['LAT_ACCEL_FACTOR']
+    tune.torque.kp = 1.0
+    tune.torque.kf = 1.0
+    tune.torque.ki = 0.1
```

Comment:
want these to be tunable per-car? if not, these should probably just be constants in latcontrol_torque.py

Reply:
Not sure yet if they need to be